tests/unit/helpers/sl/synchronous/requires-test.js

```diff
@@ -12,283 +12,141 @@ test( 'it exists', function( assert ) {
 
-    assert.ok(
-        assertionThrown,
+    // Number
+    testPropertyOne.set( 'parameter', function() {} );
+    testPropertyTwo.set( 'parameter', 123 );
+
+    assert.throws(
+        callRequires,
         'Second parameter was a number'
     );
 
     // String
-    assertionThrown = false;
+    testPropertyTwo.set( 'parameter', 'testString' );
 
-    try {
-        requires(
-            'testString',
-            'testString'
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callRequires,
         'Second parameter was a string'
     );
 
-    // Array
-    assertionThrown = false;
-
-    try {
-        requires(
-            [],
-            []
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Second parameter was an array'
-    );
-
     // Object
-    assertionThrown = false;
-
-    try {
-        requires(
-            {},
-            {}
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    testPropertyTwo.set( 'parameter', {} );
 
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callRequires,
         'Second parameter was an object'
     );
 
     // Function
-    assertionThrown = false;
-
-    try {
-        requires(
-            function() {},
-            function() {}
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    testPropertyTwo.set( 'parameter', function() {} );
 
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callRequires,
         'Second parameter was a function'
     );
 
     // Undefined
-    assertionThrown = false;
+    testPropertyTwo.set( 'parameter', undefined );
 
-    try {
-        requires(
-            undefined,
-            undefined
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callRequires,
         'Second parameter was undefined'
     );
 
     // Boolean
-    assertionThrown = false;
+    testPropertyTwo.set( 'parameter', false );
 
-    try {
-        requires(
-            true,
-            false
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
+    assert.throws(
+        callRequires,
         'Second parameter was a boolean'
     );
 
-    // Null
-    assertionThrown = false;
-
-    try {
-        requires(
-            null,
-            null
-        );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    // Function, Array
+    testPropertyTwo.set( 'parameter', [] );
 
     assert.ok(
-        assertionThrown,
-        'Second parameter was null'
+        callRequires(),
+        'First parameter was a function and second parameter and array'
```

Comment:
Function doesn't really come into play in the sense that the 2nd parameter needs to be an array, so reference to it can be removed